src/contributing/void-docs/style-guide.md

```diff
@@ -6,6 +6,19 @@ This style guide outlines the standards for contributing to the
 [mdBook](https://rust-lang-nursery.github.io/mdBook/) stored in the
 [void-docs](https://github.com/void-linux/void-docs/) repository.
 
+## General
+
+Step-by-step instructions containing 'magic' commands for copying-and-pasting
```

Comment:
How about: "Although there will always be cases where command listings are appropriate, the contents of the Handbook should be written primarily in prose"?

i use 'prose' here rather than 'English' in case people think we mean "the language of the Handbook is English".

Reply:
Maybe 'written language'? Although I may be tempted to submit PRs in verse now ;)

Reply:
> "Server not found!" I cried. "Can I not reach voidlinux.eu's Plutonian shore?"
> Quoth the Raven, "Nevermore."

More seriously, hmm, how about "written primarily in plain English"?

Reply:
sure this would work :)

or maybe we could say "written in English (or other languages as translations are added.)"?

Either way is fine, I will leave this up to you.

Reply:
Okay, changed to: "... written in English (or the relevant language
in the case of translations of the Handbook)."